examples/mapdl_tyre_performance/exec_mapdl.py

```diff
@@ -71,10 +71,10 @@ def execute(self):
         log.info(f"Executing: {cmd}")
         subprocess.run(cmd, shell=True, check=True, env=env)
 
-        log.info("End MAPDL execution script")
+        log.info("End MAPDL execution script.")
```

Comment:
Do you think we should use "Ending MAPDL execution script." here to be parallel with "Starting MAPDL execution script." earlier? Or use "-ing" verbs in both?

Reply:
Made them  "Start of..." and "End of...".